### ISSUE TITLE
[Accessibility][PivotBar] aria-posinset and aria-setsize are missing

### ISSUE THREAD

**Issue by ankitk94** [OfficeDev/office-ui-fabric-react#3697]
### Bug Report

- __Package version(s)__: 5.29.1
- __Browser and OS versions__: Edge

#### Describe the issue:

aria-posinset and aria-setsize properties are not defined for pivot bar items. Identified the issue using inspect tool with edge

### If applicable, please provide a codepen repro:

Issue also present on office fabric site

**Comment by yaananth** [comment]
FYI, this has to be a user agent bug. Browsers are responsible for setting them and narrator (AT) just reads them. There shouldn't be a need to set them programatically.

http://www.w3.org/TR/wai-aria-1.1/#dfn-user-agent 
https://www.w3.org/WAI/PF/aria-implementation/

> In the case of rich Internet applications, developers use DOM APIs to manipulate objects in the DOM tree to make them behave like interactive desktop GUI applications. In order to make a Web application understandable to assistive technologies, the user agent needs to map accessibility information from the elements in the DOM tree to the Accessibility APIs of the underlying operating system or software platform throughout the lifecycle of the application. The information needed is provided when developers use WAI-ARIA to supply semantic role, state, and property information for elements. The screen reader or other assistive technology uses the semantic information exposed via the accessibility API to provide an alternative rendering of an application that is meaningful to a user.

**Comment by KevinTCoughlin** [comment]
Closing as given the spec this should be correct, i.e. not having to set them since all nodes are present in the DOM. If that's not happening then as @yaananth mentioned it is likely a user agent bug. If the content in the PivotBar is dynamic i.e. a virtualized list then we may need to tackle this separately.

Feel free to re-open if you feel strongly that this is a gap in the library.
